src/main/java/net/minecraftforge/common/data/ForgeRecipeProvider.java

```diff
@@ -77,6 +71,7 @@ protected void buildCraftingRecipes(Consumer<FinishedRecipe> consumer)
         replace(Items.GOLD_INGOT,   Tags.Items.INGOTS_GOLD);
         replace(Items.IRON_INGOT,   Tags.Items.INGOTS_IRON);
         replace(Items.NETHERITE_INGOT, Tags.Items.INGOTS_NETHERITE);
+        replace(Items.COPPER_INGOT, Tags.Items.INGOTS_COPPER);
```

Comment:
What about those involving amethyst? Should they also be replaced/excluded as needed?

Reply:
As discussed on Discord, another PR should properly look into which items need to be replaced by tags.
For now, I did add the replacement for Amethyst as that seems to fall most in line with the other materials.